app/views/content_items/service_manual_topic.html.erb

```diff
@@ -22,7 +22,11 @@
       <%= @content_item.description %>
     </p>
 
-    <div class="js-hidden" data-module="accordion-with-descriptions">
+    <% if @content_item.visually_collapsed? %>
```

Comment:
Why are you only appling `accordion-with-descriptions` if it's collapsed? Doesn't seem obvious to me.